olm.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+RELEASE=4.4.7
+EXAMPLE=$(kustomize build examples/kustomization/tenant-lite | yq eval-all '. | [.]' | yq -o json | jq -c )
+
+operator-sdk generate bundle \
+  --package minio-operator \
+  --version 4.4.7 \
+  --deploy-dir resources/base \
+  --crds-dir resources/base/crds \
+  --manifests \
+  --metadata \
+  --output-dir bundles/$RELEASE \
+  --channels stable
+
+myenv=$EXAMPLE yq -i e '.metadata.annotations.alm-examples |= ("${myenv}" | envsubst)' bundles/$RELEASE/manifests/minio-operator.clusterserviceversion.yaml
+
+miniocontainer="minio/operator:v$RELEASE" yq -i e '.metadata.annotations.containerImage |= env(miniocontainer)' bundles/$RELEASE/manifests/minio-operator.clusterserviceversion.yaml
```

Comment:
for OLM perhaps we can move to `quay.io/minio/operator` as the default image @dvaldivia

Reply:
sure